README.md

```diff
@@ -157,6 +157,7 @@ Available `BTCPAYGEN_ADDITIONAL_FRAGMENTS` currently are:
 * [opt-save-storage-xxs](docker-compose-generator/docker-fragments/opt-save-storage-xxs.yml) will keep around 2 weeks of blocks (prune BTC for 5 GB) (lightning not supported)
 * [opt-lnd-autopilot](docker-compose-generator/docker-fragments/opt-lnd-autopilot.yml) will activate auto pilot on LND. (5 channels, 60% of allocation)
 * [opt-lnd-keysend](docker-compose-generator/docker-fragments/opt-lnd-keysend.yml) will activate keysend on LND.
+* [opt-lnd-wtclient](docker-compose-generator/docker-fragments/opt-lnd-wtclient.yml) will activate the watchtower client on LND.
```

Comment:
Active or passive watchtower?
Should specify IMO.

Reply:
What does that mean?

Reply:
As I understand it, there are two watchtower options available.
Both have the same function, but Passive returns all funds (minus miner fee) whereas Active takes a cut.

It would be useful to know which one (or both?) are activate through the fragment.

Reply:
"Reward watchtowers will be enabled in a subsequent release, though are still undergoing further testing and refinement."

Reply:
Okay so what I understand by your comment is that this fragment, by default, is an altruist watchtower.

Reply:
Correct. Tho I misstated the purpose of `wtclient.sweep-fee-rate` in my earlier post, it is simply the fees for the justice transaction the client is sending to the watchtower server. I imagine once the LND team fleshes out how reward watchtowers will operate we will have additional parameters.

Reply:
added ability to define a custom sweep fee rate.